src/ssort/_main.py

```diff
@@ -157,6 +164,8 @@ def _fmt_count_were(count):
             summary.append(f"{_fmt_count_were(unchanged)} left unchanged")
         if unsortable:
             summary.append(f"{_fmt_count_were(unsortable)} not sortable")
+        if not unsorted and not unchanged and not unsortable:
+            summary.append("No files are present to be sorted. Nothing to do.")
 
         sys.stderr.write(", ".join(summary) + "\n")
```

Comment:
Please update the tests in `test_executable.py`.

Reply:
Added a test case for this change.